src/components/Number/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Number = ({inputNumber = 0}) => (
```

Comment:
Here you will probably need a callback to notify to its parent that someone clicked on the button

Reply:
I put the callback. 

I print the number in the screen and add a `+1` in the `onPushed` function just to see how the change happend, I will delete that `+1` in the function.

But without the `+1`, is that the way that I would notify the change to the parent?, in this case, that the button was clicked.